jdaviz/configs/cubeviz/plugins/mixins.py

```diff
@@ -63,7 +63,7 @@ def _set_slice_indicator_value(self, value):
 
 
 class WithSliceSelection:
-    @cached_property
+    @property
```

Comment:
is there a reason we need to lose the caching?

Reply:
It seemed to be caching the wrong axis and this worked as a quick fix, I'd like to look into it a bit more once I resolve a few more other things.